app/javascript/api/listingApiService.ts

```diff
@@ -30,6 +30,17 @@ export const getListing = async (listingId?: string): Promise<RailsListing> =>
 export const getLotteryBucketDetails = async (listingId: string): Promise<RailsLotteryResult> =>
   get<RailsLotteryResult>(lotteryBuckets(listingId)).then((response) => response.data)
 
+/**
+ * Get the lottery buckets with rankings for the given listing, this is a workaround for slow performance on larger Lottery API payloads
+ * We don't actually need the preferenceResults data that the lotteryBuckets endpoint provides
+ * @param {string} listingId
+ * @returns {RailsLotteryResult} lottery result info
+ */
+export const getLotteryBucketDetailsWorkaround = async (
+  listingId: string
+): Promise<RailsLotteryResult> =>
+  get<RailsLotteryResult>(lotteryRanking(listingId, "000000000")).then((response) => response.data)
+
```

Comment:
If this is a temp workaround I think we can add a todo comment with a follow up ticket to clean this up in the future or remove the unused `getLotteryBucketDetails` function above.